gateway-release/src/assembly.xml

```diff
@@ -90,6 +90,13 @@
                 <exclude>**/*.bundle.map</exclude>
             </excludes>
         </fileSet>
+        <fileSet>
+            <directory>${project.build.directory}/deployments</directory>
+            <outputDirectory>data/deployments</outputDirectory>
+            <includes>
+                <include>**/%2F*/**/*</include>
```

Comment:
weird `%2F`??

Reply:
There are two additional folders here (`META-INF` and the war-plugin generated empty `WEB-INF` which should not be placed in the final `deployments` folder.